Gemfile.lock

```diff
@@ -420,6 +420,10 @@ GEM
     kaminari (0.17.0)
       actionpack (>= 3.0.0)
       activesupport (>= 3.0.0)
+    kaminari-actionview (1.0.1)
```

Comment:
These 4 lines shouldn't be here. Please remove. see https://github.com/bundler/bundler/issues/5364